lib/devise/uncommon_password.rb

```diff
@@ -3,7 +3,7 @@
 
 module Devise
   mattr_accessor :password_matches
-  @@password_matches = 100
+  @@password_matches = 200
```

Comment:
Should we have a difference between this and the config settings?

Reply:
200 is a default value and can be customized through the config.